METRICS.md

```diff
@@ -65,8 +65,11 @@ Here is an example of grafana dashboard importing.
 |:--------------------------------------:|:-----------------:|:---------------------------------------------------------------------------------------------------------------:|
 |              WorkerCount               |      master       |                                          The count of active workers.                                           |
 |          ExcludedWorkerCount           |      master       |                                     The count of workers in excluded list.                                      |
+|            ApplicationCount            |      master       |                                The count of running applications in the cluster.                                |
 |             OfferSlotsTime             |      master       |                                            The time of offer slots.                                             |
 |             PartitionSize              |      master       |          The estimated partition size of last 20 flush window whose length is 15 seconds by defaults.           |
+|            PartitionWritten            |      master       |                         The size of the data written to disk in the shuffle partition.                          |
+|           PartitionFileCount           |      master       |                                      The count of shuffle partition files.                                      |
```

Comment:
Ditto, should make it clear it's running? or total? cc @RexXiong @zwangsheng

Reply:
@AngersZhuuuu, the `PartitionWritten` and `PartitionFileCount` means the total data writen and files, which is similar with `PartitionSize`. Therefore it's better to keep the naming.